SensorFusion_src/main_test.c

```diff
@@ -0,0 +1,43 @@
+/*******************************************************
+* Author : Shyam Bhuptani 
+* Function : Unit Testing of all Functions
+*/
+
+/*Including all dependancies*/
+#include <stdio.h>
+#include <stdlib.h>
+#include "../include/Support_Degree_Matrix.h"
+#include "../include/contribution.h"
+#define num1 2
+#define num2 3
+
+// Main test function
+int main(int argc, char ** argv){
+
+    /*Testing of Function Support Degree Matrix using one-D array*/
```

Comment:
This comment is not really helpful. I suggest that you provide a more descriptive summary of the test.